src/ocean/io/select/EpollSelectDispatcher.d

```diff
@@ -800,7 +830,9 @@ public class EpollSelectDispatcher : IEpollSelectDispatcherInfo
 
                 version ( EpollCounters ) if ( n == 0 ) this.counters.timeouts++;
 
-                this.selected_set = events[0 .. n];
+                this.selected_set = this.events[0 .. n];
+                scope (exit)
+                    this.selected_set = null;
```

Comment:
I think these three lines should be in the first commit.

Reply:
Correct, I messed up the fixup.

Reply:
Forgot to mention, this is moved to the right commit.